README.md

```diff
@@ -72,6 +72,10 @@ npm run lint-css
 npm run sort-styles
 ```
 
+### Translations & Localization
+
+See [/src/locales/README.md](/src/locales/README.md)
```

Comment:
This exists a few paragraphs up.

Reply:
Woops, missed that. I reverted the file.